lib/util.js

```diff
@@ -0,0 +1,170 @@
+'use strict'
+
```

Comment:
util.js sounds like a very generic name which is prone to become a God Object. maybe it could be split into more focused smaller files?